_posts/2020-03-19-newpipe-0.18.7-released.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: "NewPipe 0.18.7 released: Hotfixing SoundCloud"
+short: "NewPipe 0.18.7 released"
+date: 2020-03-19 00:00:00 +0100
+categories: [release, pinned]
+author: Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+Yes, this is another hotfix release. Can't stop us now. In this version, SoundCloud gets fixed.<!-- more -->
+
+## Why?
+
+Up until this release, NewPipe was using an subdomain to get the data from SoundCloud called `api`. This way of accessing SoundClouds API was not working consistently anymore for all users, so the developers decided to switch to `api-v2`. While that sounds and is a change under the hood of NewPipe, there are some minor improvements even you as an end user can face.
```

Comment:
The way to check whether the `client_id` is still valid was also broken. Also, it should be "SoundCloud's" instead of "SoundClouds".